Jenkinsfile

```diff
@@ -77,14 +77,15 @@ node {
       }
     }
   }
-  
+
   cleanUp()
   slackSend channel: "#${SLACK_CHANNEL}", color: "#389a07", message: "Successfully build Instana agent docker ${INSTANA_AGENT_RELEASE} \n(<${env.BUILD_URL}|Open>)"
 }
 
 def buildImage(name, context) {
   try {
     sh """
+      cp -r ./util ./${context}/ \
```

Comment:
Copy from the generic `util` directory to the context-roots, so that we only need to maintain a single version of the `crashReport` script and don't need to copy it around ourselves.